File: examples/python/responses.py
"""Guide: https://useinfergate.com/guides/python-openai-api"""
import os
from openai import OpenAI

client = OpenAI(
    api_key=os.environ["INFERGATE_API_KEY"],
    base_url="https://api.useinfergate.com/v1",
)
response = client.responses.create(
    model=os.environ.get("INFERGATE_MODEL", "gpt-5.5"),
    input="Return a short release note for an API client update.",
    max_output_tokens=256,
)
if response.status != "completed" or not response.output_text.strip():
    raise RuntimeError("Response did not complete with useful text")
print(response.output_text)

File: examples/node/responses.mjs
// Guide: https://useinfergate.com/guides/nodejs-openai-api
import OpenAI from 'openai';

if (!process.env.INFERGATE_API_KEY) throw new Error('Set INFERGATE_API_KEY');
const client = new OpenAI({
  apiKey: process.env.INFERGATE_API_KEY,
  baseURL: 'https://api.useinfergate.com/v1',
});
const response = await client.responses.create({
  model: process.env.INFERGATE_MODEL || 'gpt-5.5',
  input: 'Return a short release note for an API client update.',
  max_output_tokens: 256,
});
if (response.status !== 'completed' || !response.output_text?.trim()) {
  throw new Error('Response did not complete with useful text');
}
console.log(response.output_text);
